samples/sensor/mcux_acmp/README.rst

```diff
@@ -26,3 +31,13 @@ ACMP input voltage by turning the on-board potentiometer.
    :board: twr_ke18f
    :goals: flash
    :compact:
+Building and Running for MIMXRT1170-EVK
```

Comment:
The lack of blank line on line 33 is likely causing the CI error here:
` /home/runner/work/zephyr/zephyr/doc/_build/src/samples/sensor/mcux_acmp/README.rst:34:Explicit markup ends without a blank line; unexpected unindent.`

Reply:
> The lack of blank line on line 33 is likely causing the CI error here: ` /home/runner/work/zephyr/zephyr/doc/_build/src/samples/sensor/mcux_acmp/README.rst:34:Explicit markup ends without a blank line; unexpected unindent.`

Thanks for identifying the issue. Fixed the blank line.